app/src/scenes/phase2/scenes/MonEquivalence/EditEquivalence.jsx

```diff
@@ -95,6 +95,16 @@ export default function EditEquivalence() {
       })();
   }, [young, id]);
 
+  const handleSelectType = (type) => {
+    setData((prevData) => ({
+      ...prevData,
+      type,
+      sousType: "",
+      missionDuration: type !== "Autre" ? 84 : prevData?.missionDuration,
```

Comment:
Je trouve que c'est plus clair dans ce sens :
```suggestion
      missionDuration: type === "Autre" ? prevData?.missionDuration : 84,
```
D'autre part, on pourrait mettre cette logique côté serveur :
- rendre missionDuration optionnel
- par défaut, la valeur qui serait enregistrée pour la durée de la mission serait 84h, mais si le client envoie une valeur pour missionDuration, on prend cette valeur à la place.
Mais comme on est dans le cadre d'une quickwin, ce que tu proposes me convient.